cmd/tt/buckets.go

```diff
@@ -203,6 +203,11 @@ File structure is mirrored in the bucket. For example, given the directory:
 
 These 'push' commands result in the following bucket structures.
 
+'tt buckets push foo/ mybuck':
+    mybuck/one.txt
+    mybuck/bar/two.txt
+		mybuck/bar/baz/three.txt
+		
```

Comment:
moved this up to the top of the list since the first thing i typically do in a new bucket is push some dir to the root of the bucket

Reply:
I think the `mybuck/` prefixes in these results should be deleted. Or maybe missing something?

Reply:
Yep, +1

Reply:
right, cool, will fix